features/_trustedboot/file.include/usr/lib/systemd/system-generators/detect-disk-by-efivars

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+set -eufo pipefail
+
+efi_part_uuid="$(cat /sys/firmware/efi/efivars/LoaderDevicePartUUID-4a67b082-0a4c-41cf-b6c7-440b29bb8c4f | tail -c +4 | tr -cd '[:print:]' | tr '[:upper:]' '[:lower:]')"
```

Comment:
Can this part change?

Reply:
same as with the secboot vars, except these are defined by the systemd / UAPI group boot loader spec. so far more likely to change than the EFI spec, but still small enough chance to bet on it being fixed.